docs/Overview.md

```diff
@@ -36,11 +32,11 @@ lambda-inference-toolkit/
 
 First models are trained and exported into model zoo, ONNX or saved as checkpoint (CKPT). Then we use Lambda Inference Toolkit to pack the model into a package that's Inference + Lambda enabled (*details on this part explained in the next graph*). Then the deployed package will be served on AWS Lambda with charge by query, reasonable response time, and effortless and limitless scalability. The serverless model serving could be triggered by AWS Services (S3, API Gateway, Kinesis, Dynamo, Kinesis, etc.), API Callbacks, HTTP, Mobile, etc., and the serving itself is also supported by AWS services (together with Java, Python, etc.).
 
-**Lambda Inference Toolkit**
+**MXNet Lambda**
 
-The Lambda Inference Toolkit has three major functions and three major components. Three major functions are **Inference Package Construction, Inference Package Maintenance and Easy Package Deployment.** These three major functions are supported by three major components: LIT Client, LIT Base Package, and LIT Model Bucket. 
+MXNet Lambda has two major functions and four major components. Three major functions are **Consuming Model Archive on AWS Lambda and  Running Inference with Apache MXNet (incubating) on AWS Lambda.** These two major functions are supported by four major components: Model Archive Support Framework, Base Code for Running Inference on AWS Lambda, SAM Templates Support, and Utilities. 
 
-LIT Base Package is a base package with decent modularization that runs a sample Lambda enabled inference task. LIT Model Bucket is an online S3 resource that host models for Lambda enabled inference. With pre-packed LIT Base Package (with MxNet pre-configured), users only need to customize local inference logics to create their own deployable Lambda enabled inference package. LIT Base Package also communicate LIT Model Bucket when being served on AWS Lambda. LIT Client is a command line tool based on AWS CLI to significantly simplify the process of construct, maintain, deploy, and update a Lambda enabled inference package. 
+Model Archive Support Framework is a framework that supports consuming Model Archive formatted machine learning models on AWS Lambda. Base Code for Running Inference on AWS Lambda is a base package with decent modularization that runs a sample Lambda enabled inference task. SAM Templates Support is AWS Serverless Application Model descriptive code to significantly simplify the process of construct, maintain, test, deploy, and update a Lambda enabled inference package. Utilities provides scripts that help process Model Archives. 
```

Comment:
Model Archive Support Framework -> Model Archive is a framework that supports consuming machine learning models

Reply:
Modified accordingly